app/configuration/Config.scala

```diff
@@ -22,4 +23,5 @@ object Config {
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$contributeUrl$uri") & idSkipConfirmation
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
 
+  val guardianDomain = ".theguardian.com"
```

Comment:
I'm not sure this is necessary, won't the host from config value above already be set to theguardian.com in prod?